doc/source/dev/dev_quickstart.rst

```diff
@@ -84,21 +84,25 @@ Next, set up your development environment.
         Your command prompt now lists the name of your new environment, like so
         ``(scipy-dev)$``.
 
-Finally, build SciPy for development and run the test suite to make sure your
-installation is successful. On Linux and OSX, you should use::
+Finally, build SciPy for development. On Linux and OSX, you should use::
 
-    python dev.py
-
-This builds SciPy first, so the first time it may take some time.
+    python dev.py build
 
 If you run into a build issue, or need more detailed build documentation
 including building on Windows, see :ref:`building`.
 
+Otherwise, if the build has finished successfully, you should now be able
+to run the SciPy tests. You can run the full test suite running::
```

Comment:
`running` -> `by running` most likely, but doesn't seem worth more turnover just for that at the moment